CartApi/Infrastructure/Exceptions/CartDomainException.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ShoesOnContainers.Services.CartApi.Infrastructure.Exceptions
```

Comment:
```suggestion
namespace EventBriteAssignment.Services.CartApi.Infrastructure.Exceptions
```
change namespace name